client/examples/js/examples/iotGetHubRuntimeInfo.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
```

Comment:
if you're expecting people to use this as an example of the js code to write, i'd write it directly in js. The typescript `__awaiter` and such are noise that distracts from the real code someone would write